registry/README.md

```diff
@@ -0,0 +1,52 @@
+# OpenTelemetry Registry Instrumentation
+
+Todo: Add a description.
```

Comment:
Whoops, missed these.  Will wait till Todos are finished before merging.

Reply:
My bad

Reply:
@robertlaurin Updated the README. Please take another look at your earliest convenience.